docs/development/RELEASE_LIFECYCLE.md

```diff
@@ -38,6 +38,27 @@ Unlike traditional GitFlow where all features eventually merge to a single integ
     - From `develop`: Independent ZetaClient release
     - From `main`: Coordinated release with ZetaCore
 
+## Protocol Contracts Integration
+
+### Repositories
+
+- https://github.com/zeta-chain/protocol-contracts-evm
+- https://github.com/zeta-chain/protocol-contracts-solana
+- https://github.com/zeta-chain/protocol-contracts-ton
+- https://github.com/zeta-chain/protocol-contracts-sui
+
+### Overview
+
+Protocol contracts repositories are related to ZetaClient versions, as ZetaClient interacts with deployed smart contracts across different chains. Each protocol contracts repository maintains its own independent versioning.
```

Comment:
also a nit, but protocol contracts evm contains gatewayZevm as well, where changes might impact zetacore, not sure if we should mention that as well